src/unix/linux.c

```diff
@@ -111,6 +115,8 @@
 #  define __NR_getrandom 359
 # elif defined(__s390__)
 #  define __NR_getrandom 349
+# elif defined(__riscv)
+#  define __NR_getrandom 384
```

Comment:
Are you sure this is right? I thought getrandom was 278 and statx 291. copy_file_range is indeed 285.

Reply:
You are right - fixed.